### PR TITLE
Added regression tests for Migrator

### PR DESCRIPTION
**Pull request by emystein**
MigratorTest run against embedded Cassandra.
CassandraVersionerTest run against mocked Session object.

### CODE REVIEW

**Comment by nbozic** [comment]
Can you please merge with changes I made to `travis.yml`.

Embedded Cassandra process took too long so I had to change it to use sudo:true in order not to timeout. That merge should fix travis build as well of this commit.

File: pom.xml [hunk 1]
```diff
@@ -16,6 +16,7 @@
         <guava.version>16.0.1</guava.version>
         <slf4j.version>1.7.7</slf4j.version>
         <junit.version>4.9</junit.version>
+        <mockito.version>2.0.52-beta</mockito.version>
```
Comment (nbozic) [inline pom.xml]: Why use beta version, do we need any special feature of this beta?

Comment (emystein) [inline pom.xml]: It's the latest version from the Maven release repository. Even if has "beta" in its name, is a release. We can go back to the previous version without "beta", I think is 1.x.

File: src/test/java/io/smartcat/migration/CassandraMetadataAnalyzer.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import com.datastax.driver.core.Metadata;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.TableMetadata;
+import com.google.common.base.Optional;
+
+public class CassandraMetadataAnalyzer {
+
+    private Session session;
+
+    public CassandraMetadataAnalyzer(Session aSession) {
```
Comment (mgobec) [inline src/test/java/io/smartcat/migration/CassandraMetadataAnalyzer.java]: Please use standard naming convention for session parameter and 'this' for private field access.

**Comment by mgobec** [comment]
@emystein Thanks for this update. Could you please rebase with develop because we updated travis.yml so that the builds pass?

**Comment by mgobec** [comment]
@emystein Thanks I'm merging this

**Pull request merged**